sql/parse/parse.go

```diff
@@ -193,12 +193,12 @@ func tableExprsToTable(te sqlparser.TableExprs) (sql.Node, error) {
 		return nodes[0], nil
 	}
 
-	if len(nodes) == 2 {
-		return plan.NewCrossJoin(nodes[0], nodes[1]), nil
+	join := plan.NewCrossJoin(nodes[0], nodes[1])
```

Comment:
should we check the len of nodes before? (or use left, right)

Reply:
nevermind: https://github.com/src-d/go-mysql-server/pull/49/files#diff-913eb5b14dd573d980e4f89476047ab6R192